src/Neo/SmartContract/ApplicationEngine.cs

```diff
@@ -253,7 +253,9 @@ protected static void OnSysCall(ExecutionEngine engine, Instruction instruction)
         /// <param name="datoshi">The amount of GAS, in the unit of datoshi, 1 datoshi = 1e-8 GAS, to be added.</param>
         protected internal void AddFee(long datoshi)
         {
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Wont this disable the warning on user side as well?

Reply:
No